interactive_engine/executor/store/groot/src/api/condition/mod.rs

```diff
@@ -13,15 +13,30 @@
 //! See the License for the specific language governing permissions and
 //! limitations under the License.
 
-use super::property::*;
-use crate::schema::prelude::*;
+mod oprand;
+pub mod predicate;
+#[cfg(test)]
+mod test;
+pub use oprand::Oprand;
+pub use predicate::NormalOperator;
+pub use predicate::Predicate;
+
+use super::filter::ElemFilter;
+use super::{Edge, Vertex};
+use crate::GraphResult;
 
 #[derive(Debug, Clone)]
 pub enum Condition {
     And(AndCondition),
     Or(OrCondition),
     Not(NotCondition),
-    Cmp(CmpCondition),
+    Pred(Predicate),
```

Comment:
I think the original name of `Cmp(CmpCondition)` may be a better choice?

Reply:
I prefer `Pred` because it also contains `HasProp`.

Reply:
rename `Predicate` to `PredCondition`